types/static.d.ts

```diff
@@ -1,59 +0,0 @@
-/* Use this file to declare any custom file extensions for importing */
```

Comment:
Isn't this generated by Vite? And provides TS support for these kinds of files?

Reply:
No idea